src/index.html

```diff
@@ -12,7 +12,42 @@
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__img">
```

Comment:
no needs to wrap `img` with `div`. Apply styles directly to `img` class selector